lib_xua/src/core/endpoint0/xua_ep0_descriptors.h

```diff
@@ -584,9 +599,22 @@ unsigned char devQualDesc_Null[] =
 #define HS_STREAM_FORMAT_OUTPUT_2_MAXPACKETSIZE (MAX_PACKET_SIZE_MULT_OUT_HS * HS_STREAM_FORMAT_OUTPUT_2_SUBSLOT_BYTES)
 #define HS_STREAM_FORMAT_OUTPUT_3_MAXPACKETSIZE (MAX_PACKET_SIZE_MULT_OUT_HS * HS_STREAM_FORMAT_OUTPUT_3_SUBSLOT_BYTES)
 
-#if (HS_STEAM_FORMAT_OUPUT_1_MAXPACKETSIZE > 1024) || (HS_STEAM_FORMAT_OUPUT_2_MAXPACKETSIZE > 1024) \
-    || (HS_STEAM_FORMAT_OUPUT_3_MAXPACKETSIZE > 1024)
-#error
+#if (HS_STREAM_FORMAT_OUTPUT_1_MAXPACKETSIZE > 1024)
```

Comment:
I think this useful fix should be mentioned in the changelog.

Reply:
done